### PR TITLE
Curate domains for all remaining ontologies

### PR DESCRIPTION
**Pull request by cthoyt**
It took about 25 minutes to go through and give a good shot at annotating all of the remaining ontologies that didn't have one. I just read through the descriptions, some of them that had replacement annotations were even easier since I could just grab the domain from the updated one.

By definition all of the remaining ontologies are orphaned, deprecated, or obsolete, so there is no bureaucratic need to ask anyone if we can update these metadata, but any suggestions are welcome.

### CODE REVIEW

File: ontology/ev.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ title: eVOC (Expressed Sequence Annotation for Humans)
 contact:
   email: evoc@sanbi.ac.za
   label: eVOC mailing list
+domain: chemistry and biochemistry
```
Comment (cmungall) [inline ontology/ev.md:8]: Should be anatomy and development

File: ontology/ogi.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ contact:
   label: Asiyah Yu Lin
   orcid: 0000-0002-5379-5359
 description: An ontology that formalizes the genomic element by defining an upper class genetic interval
+domain: investigations
```
Comment (cmungall) [inline ontology/ogi.md:11]: Should be the same as SO

**Review by cmungall — APPROVED** [review]

File: ontology/ev.md (hunk 1 above)
Comment (cthoyt) [reply ontology/ev.md:8]:
```suggestion
domain: anatomy and development
```

File: ontology/ogi.md (hunk 2 above)
Comment (cthoyt) [reply ontology/ogi.md:11]:
```suggestion
domain: chemistry and biochemistry
```

Should OGSF also be updated then?

**Comment by cthoyt** [comment]
I made the changes suggested by Chris. this should be ready to merge

**Review by matentzn — APPROVED** [review]
Excellent detective work and great effort. I have reviewed every single ontology, and while some of them are a bit ambiguous, I also couldn't find a better domain to fit them in.

**Pull request merged**